From cd2267fe5cb54933b0b409d966da9313ace99bcb Mon Sep 17 00:00:00 2001
Message-Id: <cd2267fe5cb54933b0b409d966da9313ace99bcb.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Thu, 17 Jul 2014 13:32:30 +0530
Subject: [PATCH 201/312] FOR_UPSTREAM [VPG] drm/i915: Fix printing proper
 min/min/rpe values in debugfs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This was fumbled while trying to use the cached min/min/rpe values in the vlv
debugfs code.

This is a regression from

commit 03af20458a57a50735b12c1e3c23abc7ff70c6fa
Author: Ville Syrjälä <ville.syrjala@linux.intel.com>
Date:   Sat Jun 28 02:03:53 2014 +0300

    drm/i915: Use the cached min/min/rpe values in the vlv debugfs code

Change-Id: I0c6da333ae14d36a93582b281603fa24f1b95dcf
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_debugfs.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_debugfs.c b/drivers/gpu/drm/i915/i915_debugfs.c
index 8007c3d..2ddab13 100644
--- a/drivers/gpu/drm/i915/i915_debugfs.c
+++ b/drivers/gpu/drm/i915/i915_debugfs.c
@@ -1131,13 +1131,13 @@ static int i915_frequency_info(struct seq_file *m, void *unused)
 		seq_printf(m, "DDR freq: %d MHz\n", dev_priv->mem_freq);
 
 		seq_printf(m, "max GPU freq: %d MHz\n",
-			   dev_priv->rps.max_freq);
+			   vlv_gpu_freq(dev_priv, dev_priv->rps.max_freq));
 
 		seq_printf(m, "min GPU freq: %d MHz\n",
-			   dev_priv->rps.min_freq);
+			   vlv_gpu_freq(dev_priv, dev_priv->rps.min_freq));
 
 		seq_printf(m, "efficient (RPe) frequency: %d MHz\n",
-			   dev_priv->rps.efficient_freq);
+			   vlv_gpu_freq(dev_priv, dev_priv->rps.efficient_freq));
 
 		seq_printf(m, "current GPU freq: %d MHz\n",
 			   vlv_gpu_freq(dev_priv, (freq_sts >> 8) & 0xff));
-- 
1.7.9.5

